### PR TITLE
Add verbosity levels to lsif upload

### PR DESCRIPTION
**Pull request by efritz**
Add trace logging to lsif upload actions.

### CODE REVIEW

File: cmd/src/lsif_upload.go [hunk 1]
```diff
@@ -67,6 +70,7 @@ Examples:
 	flags.maxPayloadSizeMb = flagSet.Int("max-payload-size", 100, `The maximum upload size (in megabytes). Indexes exceeding this limit will be uploaded over multiple HTTP requests.`)
 	flags.ignoreUploadFailures = flagSet.Bool("ignore-upload-failure", false, `Exit with status code zero on upload failure.`)
 	flags.uploadRoute = flagSet.String("upload-route", "/.api/lsif/upload", "The path of the upload route.")
+	flags.rawVerbosity = flagSet.Int("v", 0, "-v=0 shows no logs; -v=1 shows requests and response metadata; -v=2 shows headers, -v=3 shows response body")
```
Comment (eseliger) [inline cmd/src/lsif_upload.go:73]: is there no issue with this subcommand defining a "v" flag, and the main command doing so, too?

**Review by eseliger — APPROVED** [review]

File: cmd/src/lsif_upload.go (hunk 1 above)
Comment (efritz) [reply cmd/src/lsif_upload.go:73]: Does the main command currently define this flag?

Comment (eseliger) [reply cmd/src/lsif_upload.go:73]: Yes it does. Okay, seems fine then.

 I think this is possible now, but doesn't matter I think: 

`src -v lsif upload -v=-1` :D

Comment (efritz) [reply cmd/src/lsif_upload.go:73]: Ah. I'll rename to avoid the confusion.

**Pull request merged**